frameworks/Kotlin/ktor/benchmark_config.json

```diff
@@ -2,15 +2,15 @@
   "framework": "ktor",
   "tests": [
     {
-      "default": {
+      "netty": {
```

Comment:
The `default` test is the one that gets run.:

`tfb --test ktor` - runs the default test
`tfb --test ktor-netty` - runs the netty test in this case
`tfb --test-dir Kotlin/ktor` - would run all the tests in your benchmark config